### PR TITLE
Changelog 1.10.3

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Comment by 6543** [comment]
@lafriks here it is

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Fix dump non-exist log directory (#9818) (#9820)
+  * Fix compare (#9808) (#9815)
+  * Fix missing msteam webhook on organization (#9781) (#9795)
+  * Fix add team on collaborator page when same name as organization (#9783)
+  * Hide credentials when submitting migration (#9102) (#9704)
+  * Fix cache problem on dashboard (#9358) (#9703)
+  * Send tag create and push webhook when release created on UI (#8671) (#9702)
+  * Branches not at ref commit ID should not be listed as Merged (#9614) (#9639)
+* SECURITY
```
Comment (lafriks) [inline CHANGELOG.md]: move security before bugfixes as it is more important

**Comment by lafriks** [comment]
Please also add #9831 and #9835

File: CHANGELOG.md (hunk 1 above)
Comment (6543) [reply CHANGELOG.md]: done

**Comment by 6543** [comment]
@lafriks done

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,25 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix dump non-exist log directory (#9818) (#9820)
+  * Fix compare (#9808) (#9815)
+  * Fix missing msteam webhook on organization (#9781) (#9795)
+  * Fix add team on collaborator page when same name as organization (#9783)
+  * Hide credentials when submitting migration (#9102) (#9704)
+  * Fix cache problem on dashboard (#9358) (#9703)
+  * Send tag create and push webhook when release created on UI (#8671) (#9702)
+  * Branches not at ref commit ID should not be listed as Merged (#9614) (#9639)
+* MISC
+  * Never allow an empty password to validate (#9682) (#9684)
```
Comment (guillep2k) [inline CHANGELOG.md]: This looks like a security fix.

Comment (6543) [reply CHANGELOG.md]: @lunny can you tag this issue correctly ? :arrow_up:  
I'll move this

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.10.3](https://github.com/go-gitea/gitea/releases/tag/v1.10.3) - 2020-01-17
+* SECURITY
+  * Hide credentials when submitting migration (#9102) (#9704)
+  * Never allow an empty password to validate (#9682) (#9684)
+  * Prevent redirect to Host (#9678) (#9680)
+  * Hide public repos owned by private orgs (#9609) (#9616)
+* BUGFIXES
+  * Allow assignee on Pull Creation when Issue Unit is deactivated (#9836) (#9838)
+  * Fix download file wrong content-type (#9825) ()#9835)
```
Comment (lafriks) [inline CHANGELOG.md]:
```suggestion
  * Fix download file wrong content-type (#9825) (#9835)
```

Comment (6543) [reply CHANGELOG.md]: my bad ... @lafriks can you cancle drone task I'll update

**Comment by lafriks** [comment]
looks like  #9838 also needs to be added

File: CHANGELOG.md (hunk 3 above)
Comment (lafriks) [reply CHANGELOG.md]: done

**Comment by 6543** [comment]
@lafriks already in

**Review by lafriks — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Review by jolheiser — APPROVED** [review]

**Comment by techknowlogick** [comment]
@6543 please front port x2

**Comment by 6543** [comment]
@techknowlogick done

**Pull request merged**